files/bootstrap.sh

```diff
@@ -113,7 +113,7 @@ kubectl config \
 
 MAC=$(curl -s http://169.254.169.254/latest/meta-data/network/interfaces/macs/ -s | head -n 1 | sed 's/\/$//')
 CIDRS=$(curl -s http://169.254.169.254/latest/meta-data/network/interfaces/macs/$MAC/vpc-ipv4-cidr-blocks)
-TEN_RANGE=$(echo $CIDRS | grep -c '^10\..*' || true )
+TEN_RANGE=$(echo $CIDRS | grep -cE '[^\d]?10\.[\.\d\/]*' || true )
```

Comment:
This would match `210.50.112.0/24` as well, because of the `\d`.

Reply:
Oh, matching non-digits...

Reply:
I did test it first ;)

[regexr.com](https://regexr.com) is quite useful for such tasks.

A nice improvement would be to use a 'look behind' so that the space in front of the second 10.x CIDR isn't included in the match. For the purpose of this use-case, it's unnecessary (and I'm not sure Grep extended re supports it), but could be considered the cherry on the cake.

Reply:
Yeah, the non-digit doesn't really work since you have the ? (0 or 1). Examples that should not match:
```
> echo "210.50.112.0/24 99.20.112.0/24" | grep -cE '[^\d]?10\.[\.\d\/]*'     
1
> echo "220.50.112.0/24 99.10.112.0/24" | grep -cE '[^\d]?10\.[\.\d\/]*'   
1
```

Reply:
Good catch!